README.md

```diff
@@ -3,56 +3,92 @@
 ![](https://travis-ci.com/UCLA-CS130/Mr.-Robot-et-al..svg?token=yz9yBJgd4Sxya2e1weD1&branch=master)
 
 :zap: Lightning is a modern, robust web server written in C++.
-# Setup
 
-### Supported systems
 
-[Ubuntu 14.04](http://releases.ubuntu.com/14.04/)
+## Supported systems
 
+[Ubuntu 14.04 (Trusty)](http://releases.ubuntu.com/14.04/)
 
-### Installation
+
+## Installation
+
+Ensure that [git](https://www.digitalocean.com/community/tutorials/how-to-install-git-on-ubuntu-14-04) and [Docker](https://store.docker.com/editions/community/docker-ce-server-ubuntu?tab=description) are both installed:
+
+Then grab a copy of this repository:
+
+```
+$ git clone --recursive https://github.com/UCLA-CS130/Mr.-Robot-et-al..git
+```
+
+Then, to build a Lightning binary (using `Dockerfile`: 
 
 ```
-$ git clone --recursive git@github.com:UCLA-CS130/Mr.-Robot-et-al..git
+make build
 ```
 
-### Dependencies
+And to start the Lightning server (using `Dockerfile.run`):
 
-We have the following dependencies:
+```
+make deploy
+```
+
+
+### Manually Install Dependencies
+
+If you wish to avoid the use of Docker and would prefer to manually install
+everything, we have the following dependencies:
 
 * Boost: used for socket networking as well as string manipulation.
-* Lcov: used to create test coverage webpages.
-* Python 3.6: Our integration test that uses methods not supported in version of Python earlier than 3.6
-* httpie: a handy tool for checking http connections that we use in our integration test.
+* lcov: used to create test coverage webpages.
+* Python 3.6: Our integration test that uses methods not supported in version
```

Comment:
This should be take out to be consistent with our integration test. We are not using Python 3.6 anymore.